core/src/main/java/hudson/DNSMultiCast.java

```diff
@@ -77,16 +84,102 @@ public Object call() {
 
     public void close() {
         if (jmdns!=null) {
-//            try {
+            try {
                 jmdns.abort();
                 jmdns = null;
-//            } catch (final IOException e) {
-//                LOGGER.log(Level.WARNING,"Failed to close down JmDNS instance!",e);
-//            }
+            } catch (final IOException e) {
+                LOGGER.log(Level.WARNING,"Failed to close down JmDNS instance!",e);
+            }
         }
     }
 
     private static final Logger LOGGER = Logger.getLogger(DNSMultiCast.class.getName());
 
-    public static boolean disabled = SystemProperties.getBoolean(DNSMultiCast.class.getName()+".disabled");
+    public static boolean disabled = SystemProperties.getBoolean(DNSMultiCast.class.getName()+".disabled", true);
+
+    private static class JenkinsJmDNS extends JmDNSImpl {
+        private static Logger logger = Logger.getLogger(JmDNSImpl.class.getName());
+        private final Class parent;
+
+        /**
+         * Create an instance of JmDNS and bind it to a specific network interface given its IP-address.
+         *
+         * @param address IP address to bind to.
+         * @param name    name of the newly created JmDNS
+         * @throws IOException
+         */
+        public JenkinsJmDNS(InetAddress address, String name) throws IOException {
+            super(address, name);
+            this.parent = this.getClass().getSuperclass();
+        }
+
+        /**
+         * Works like {@link #close()} but terminate uncleanly, but rapidly and without blocking.
+         */
+        public void abort() throws IOException {
```

Comment:
Since we've basically copy/pasted the code from the `close` method, it might be a good idea to keep the commented out code from the fork, just in case the parent gets updated as in a future version?